packages/thumbprint-tokens/src/tokens/color.json

```diff
@@ -4,212 +4,350 @@
     "tokens": [
         {
             "id": "blue-100",
-            "value": { "web": "#eaf6fa" },
+            "value": {
+                "web": "#eaf6fa",
+                "ios": "UIColor = UIColor(red: 0.91764706, green: 0.9647059, blue: 0.98039216, alpha: 1.0)"
+            },
             "type": "color"
         },
         {
             "id": "blue-200",
-            "value": { "web": "#b3ebff" },
+            "value": {
+                "web": "#b3ebff",
+                "ios": "UIColor = UIColor(red: 0.7019608, green: 0.92156863, blue: 1.0, alpha: 1.0)"
+            },
             "type": "color"
         },
         {
             "id": "blue-300",
-            "value": { "web": "#79d2f2" },
-            "type": "color"
+            "value": {
+                "web": "#79d2f2",
+                "ios": "UIColor = UIColor(red: 0.4745098, green: 0.8235294, blue: 0.9490196, alpha: 1.0)"
+            },
+            "type": "color",
+            "private": true
```

Comment:
What's the `"private": true` property on some of the colors?

Reply:
Whoops. Added that by accident. It's something we used to have in our tokens set-up for colors that we wanted to reserve for use only within Thumbprint. We've since gotten rid of that concept.